### DISCUSSION TITLE
Ram problem

### DISCUSSION THREAD

**Discussion by Diamondstudioo** [IzzelAliz/Arclight#1090] [Q&A]
Why when I put arclight my server consumes too much ram, it consumes more than 8GB

**Comment by chocheger** [comment]
how many mods do u have? in my server consumes like 9gb but we have 260 mods, 6 plugins, 1.19.2

**Comment by Zhou-Shilin** [comment]
use `java -Xmx8G -jar <server-jar-file>.jar nogui`
replace `8G` to the max ram u expect arclight to use

**Reply by Skillbombe** [reply]
dosent work
